src/Command/InfectionCommand.php

```diff
@@ -122,14 +122,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->container['infection.config'] = function (Container $c) use ($input): InfectionConfig {
             try {
-                $configPaths = [
-                    InfectionConfig::CONFIG_FILE_NAME,
-                    InfectionConfig::CONFIG_FILE_NAME . '.dist',
-                ];
+                $configPaths = [];
                 $customConfigPath = $input->getOption('configuration');
                 if ($customConfigPath) {
                     $configPaths[] = $customConfigPath;
                 }
+                $configPaths = [
```

Comment:
if I'm not mistaken `$configPaths` with `$customConfigPath` will be overridden by
```
 $configPaths = [
...
];
```
So in this case $customConfigPath won't work as expected as it'll be excluded.

Reply:
just wanted to write the same ;)